subsys/bluetooth/audio/broadcast_sink.c

```diff
@@ -41,6 +41,26 @@ static sys_slist_t sink_cbs = SYS_SLIST_STATIC_INIT(&sink_cbs);
 
 static void broadcast_sink_cleanup(struct bt_audio_broadcast_sink *sink);
 
+static void broadcast_sink_clear_big(struct bt_audio_broadcast_sink *sink)
+{
+	sink->big = NULL;
+	sink->stream_count = 0;
+	sink->streams = NULL;
+}
+
+static struct bt_audio_broadcast_sink *broadcast_sink_get_iso_chan(const struct bt_iso_chan *chan)
```

Comment:
I would suggest different name e.g. `broadcast_sink_lookup_iso_chan` because it looks up for bt_audio_broadcast_sink based on bt_iso_chan.
Similarly to `bt_conn_lookup_handle` or `bt_conn_lookup_addr_le`.

For me `broadcast_sink_get_iso_chan` sounds like it returns `iso_chan` for given `broadcast_sink`

Reply:
Agreed